.travis.yml

```diff
@@ -49,8 +49,6 @@ before_install:
   - if [ "$SPEEDUPS" == "1" ]; then pip install --install-option="--no-cython-compile" cython; fi
   # if testing without numpy explicitly remove it
   - if [ "$NUMPY" == "0" ]; then pip uninstall --yes numpy; fi
-  # convert SPEEDUPS to --with-speedups/--without-speedups
-  - if [ "$SPEEDUPS" == "1" ]; then SPEEDUPS_FLAG=--with-speedups; else SPEEDUPS_FLAG=--without-speedups; fi
```

Comment:
Yes, certainly. But since we are probably going to refactor this completely (eg moving to github actions), similarly as we did in pygeos, I would maybe leave further clean-up for that change?

Reply:
Sure, that can wait until then. And yes, I was thinking that we may want to refactor the CI setup based on the recent pygeos work.